### ISSUE TITLE
Vector image missing a text alternative - Iowa Health Care Logo

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9672]
# Motivation

Siteimprove is reporting "Vector image missing a text alternative". This is a WCAG 2.1 A compliance issue related to the Iowa Health Care logo in both the header and footer.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r43

Siteimprove link to existing issue
- https://my2.siteimprove.com/Inspector/1215296/A11y/Page?pageId=73785440477&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r43/failed
- https://my2.siteimprove.com/Inspector/1243662/A11y/Page?pageId=88371083767&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r43/failed/bU80TGtQcWE=;YjdiOGZiN2E=

Example
- https://cdd.center.uiowa.edu/about-cdd
- https://holden.uihealthcare.org/

**Vector image missing a text alternative**
Issue
1.1.1: Non-text Content
All non-decorative images should have a descriptive text alternative.
